src/java/org/apache/cassandra/db/compaction/CompactionStrategyManager.java

```diff
@@ -610,6 +617,32 @@ public long[] getPerLevelSizeBytes()
         }
     }
 
+    public int[] getSSTableCountPerTWCSBucket()
+    {
+        readLock.lock();
+        try
+        {
+            List<Map<Long, Integer>> countsByBucket = Stream.concat(
+                                                            StreamSupport.stream(repaired.allStrategies().spliterator(), false),
+                                                            StreamSupport.stream(unrepaired.allStrategies().spliterator(), false))
```

Comment:
nit: Any way we can indent the concatenated streams here, so it's easier to read onward to `filter`?

Reply:
Ew. Good catch.